src/tests/nativeaot/SmokeTests/HelloWasm/HelloWasm.cs

```diff
@@ -444,6 +446,25 @@ private static unsafe void TestStructStoreWithSignificantPadding()
         EndTest(*ptrCopySized == 2);
     }
 
+    struct LlvmStruct
+    {
+        internal int i1, i2;
+    }
+
+    // dont want this inlined as it wi the `LCL_VAR_ADDR`
+    [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
Most codegen tests mark methods as noinline for similar reasons.
```suggestion
    [MethodImpl(MethodImplOptions.NoInlining)]
```